eng/build.ps1

```diff
@@ -20,46 +20,78 @@ Param(
-  Write-Host "  -os                       Build operating system: Windows_NT, Linux, OSX, or Browser"
-  Write-Host "  -arch                     Build platform: x86, x64, arm, arm64, or wasm (short: -a). Pass a comma-separated list to build for multiple architectures."
-  Write-Host "  -configuration            Build configuration: Debug, Release or [CoreCLR]Checked (short: -c). Pass a comma-separated list to build for multiple configurations"
-  Write-Host "  -runtimeConfiguration     Runtime build configuration: Debug, Release or [CoreCLR]Checked (short: -rc)"
-  Write-Host "  -librariesConfiguration   Libraries build configuration: Debug or Release (short: -lc)"
-  Write-Host "  -verbosity                MSBuild verbosity: q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic] (short: -v)"
-  Write-Host "  -binaryLog                Output binary log (short: -bl)"
-  Write-Host "  -help                     Print help and exit (short: -h)"
+  Write-Host "  -arch (-a)                     Build platform: x86, x64, arm, arm64, or wasm."
+  Write-Host "                                 Pass a comma-separated list to build for multiple architectures."
+  Write-Host "                                 [Default: Your machine's architecture.]"
+  Write-Host "  -binaryLog (-bl)               Output binary log."
+  Write-Host "  -configuration (-c)            Build configuration: Debug, Release or [CoreCLR]Checked."
+  Write-Host "                                 Pass a comma-separated list to build for multiple configurations."
+  Write-Host "                                 [Default: Debug]"
+  Write-Host "  -help (-h)                     Print help and exit."
+  Write-Host "  -librariesConfiguration (-lc)  Libraries build configuration: Debug or Release."
+  Write-Host "                                 [Default: Debug]"
+  Write-Host "  -os                            Build operating system: Windows_NT, Linux, OSX, or Browser."
+  Write-Host "                                 [Default: Your machine's OS.]"
+  Write-Host "  -runtimeConfiguration (-rc)    Runtime build configuration: Debug, Release or [CoreCLR]Checked."
+  Write-Host "                                 [Default: Debug]"
+  Write-Host "  -subset (-s)                   Build a subset, print available subsets with -subset help."
+  Write-Host "                                 '-subset' can be omitted if the subset is given as the first argument."
+  Write-Host "                                 [Default: Builds the entire repo.]"
+  Write-Host "  -verbosity (-v)                MSBuild verbosity: q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic]."
+  Write-Host "                                 [Default: Minimal]"
+  Write-Host "  -vs                            Open the solution with VS using the locally acquired SDK. Path or solution name."
```

Comment:
```suggestion
  Write-Host "  -vs                            Open the solution with Visual Studio using the locally acquired SDK. Path or solution name."
```
Does this work with .csproj or only .sln @ViktorHofer  ?

Reply:
You can use it to open anything that works inside VS :)

Reply:
OK then we should say solution or project file.